boards/common/remote/include/cfg_clk_default.h

```diff
@@ -27,10 +27,40 @@ extern "C" {
 #endif
 
 /**
- * @name Clock system configuration
+ * @name    Clock system configuration
  * @{
  */
-#define CLOCK_CORECLOCK     (16000000U) /* 16MHz */
+/*
+ * 0: use internal 32KHz RCOSC
+ * 1: use external 32KHz XOSC
+ */
+#ifndef SYS_CTRL_OSC32K_USE_XTAL
+#define SYS_CTRL_OSC32K_USE_XTAL        (1)
+#endif
+/*
+ * 0: use internal 16MHz RCOSC
+ * 1: use external 32MHz XOSC, required for RF operation
+ */
+#ifndef SYS_CTRL_OSC_USE_XTAL
+#define SYS_CTRL_OSC_USE_XTAL           (1)
+#endif
+
+#if SYS_CTRL_OSC_USE_XTAL
+#define CLOCK_OSC           (XOSC32M_FREQ)
+#else
+#define CLOCK_OSC           (RCOSC16M_FREQ)
+#endif
+
+#if SYS_CTRL_OSC32K_USE_XTAL
+#define CLOCK_OSC32K        (32768U)    /* XCOSC frequency */
+#else
+#define CLOCK_OSC32K        (32753U)    /* RCOSC frequency */
```

Comment:
Should be a define (`RCOSC32K_FREQ`) just like `RCOSC16M_FREQ`

Reply:
addressed!